packages/api-explorer/src/components/Header/ApiSpecSelector.tsx

```diff
@@ -55,15 +54,6 @@ export const ApiSpecSelector: FC<ApiSpecSelectorProps> = ({
   }
 
   return (
-    <FieldSelect
-      defaultValue={spec.key}
-      options={options}
-      aria-label="Version"
-      label="Version"
-      mb="0"
-      mr="large"
-      onChange={handleChange}
-      inline
-    />
+    <Select defaultValue={spec.key} options={options} onChange={handleChange} />
```

Comment:
![Screen Shot 2020-09-01 at 11 51 32](https://user-images.githubusercontent.com/19397662/91841029-8cf73380-ec49-11ea-9fde-cd4792b22cca.png)

Was omitting the `Version` label intentional here? Unfortunately it breaks the unit test. `FieldSelect` was used because it has an associated label prop but if there's a preference for `Select` we can create a separate label here.

Reply:
@josephaxisa - It was, the FieldSelect that was there before was causing odd spacing issues in the header due to the inline attribute that places the label before the select. What if we just put Version before 4.0 in the Select? So in the example above, it would read Version 4.0 (experimental)